tools/lsp/language/properties.rs

```diff
@@ -1386,7 +1386,12 @@ component MainWindow inherits Window {
 }
             "#.to_string());
         let file_url = url.clone();
-        let (_, result) = properties_at_position_in_cache(28, 15, &mut dc, &url).unwrap();
+
+        let doc = dc.documents.get_document(&crate::language::uri_to_file(&url).unwrap()).unwrap();
+        let source = &doc.node.as_ref().unwrap().source_file;
+        let (l, c) = source.line_column(source.source().unwrap().find("base2 :=").unwrap());
+        let (_, result) =
+            properties_at_position_in_cache(l as u32, c as u32, &mut dc, &url).unwrap();
```

Comment:
This change in the test here is just searching through the code instead of using hard-coded value.
And it find the starting position of the element with an identifier (`base2 :=`) so that we have a test that when the cursor is at the beginning of an identifier we get the right properties.

The goal overall is to make sure that the outline, the properties, and the highlighted item agree on the range of the elements.